api/v1beta1/ovncontroller_types.go

```diff
@@ -128,7 +128,9 @@ type OVSExternalIDs struct {
 	// OvnEncapType - geneve or vxlan
 	OvnEncapType string `json:"ovn-encap-type"`
 
-	EnableChassisAsGateway bool `json:"enable-chassis-as-gateway,omitempty" optional:"true"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=true
+	EnableChassisAsGateway bool `json:"enable-chassis-as-gateway,omitempty"`
```

Comment:
current operator-linters don't allow using omitempty with default value assigned, can you drop that?